packages/live-share-inking/src/core/Utils.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+import { v4 as uuid } from "uuid";
+import { IPointerPoint } from "./Geometry";
+
+const EPSILON = 0.000001;
+
+/**
+ * @hidden
```

Comment:
> @hidden [](http://example.com/codeflow?start=2&length=8)

I'm personally ok with you just marking individual items with @hidden but if you look at the other packages I have a separate `internals` folder that contains code that is truly hidden.  Stuff in `internals` can be consumed by other components in the package but isn't exported from the top level index.ts.  

Worth considering...

Reply:
There's a lot of @hidden stuff in the core SDK (unless it has changed recently?) so I thought that was how you guys liked to do things. But I'll make this change, it's a lot better.

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512